sources/ham/tests/ham-test-thrift/src_client/_build.ham

```diff
@@ -1,4 +1,4 @@
-if [ SubDirOnce TOP sources ham tests ham-test-thrift src_client ] = 1 { return ; }
+if [ SubDirOnce TOP src_client ] = 1 { return ; }
```

Comment:
Btw these are necessary only if there's some circular SubInclude going on... Which really shouldn't be a thing ideally. Its sort of kludge I used in niLang etc so that you could build "everything that niUI/whatever depends on" from within the sources/niUI/src folder. But it actually doesn't make much sense.

I think ultimately what we'd want to do is remove SubDirOnce entierly, and disallow SubInclude outside of the _build.ham that link sub projects - and certainly no allow SubInclude SOMETHING_IN_A_PARENT.

Anyway, this is a note for future cleanup of the ham rules...